deps/rabbit/test/backing_queue_SUITE.erl

```diff
@@ -104,6 +113,7 @@ init_per_group(Group, Config) ->
     end.
 
 init_per_group1(backing_queue_tests, Config) ->
+    %% @todo Is that test still relevant?
```

Comment:
I'm not sure why this `@todo` is here.

Reply:
It's because I am not sure that the get_env can return anything other than rabbit_priority_queue. Perhaps the test suite was running with other modules before but it hasn't for a while. I haven't had time to check.

Reply:
Got it, I'll do some digging.